implementations/rust/ockam/ockam_command/src/identity/show.rs

```diff
@@ -51,14 +51,14 @@ fn run_impl(opts: CommandGlobalOpts, cmd: ShowCommand) -> crate::Result<()> {
     if cmd.full {
         let identity = state.config.change_history.export()?;
         if Some(EncodeFormat::Hex) == cmd.encoding {
-            print_output(identity, &opts.global_args.output_format)?;
+            println_output(identity, &opts.global_args.output_format)?;
```

Comment:
This `fn` is missing it's import: 

```rs
use crate::util::println_output;
```

This change also removes any usages of `print_output` and that associated fn can be removed too.

Reply:
Thanks for the update.

I'm running the automated actions, I do believe the rust build will warn about `fn print_ouput` definition.

I can point you in that right direction if that is the case.

Reply:
This is the error the build is running into:

https://github.com/build-trust/ockam/actions/runs/4760039036/jobs/8474497357?pr=4737#step:9:47


This is the `fn` in reference:

https://github.com/tarunsamanta2k20/ockam/blob/02323a58b856b9e6474642327d24738a68af38d4/implementations/rust/ockam/ockam_command/src/util/mod.rs#L337-L344

You can go ahead and delete that function, when you run `cargo build` in the root directory. The warning should no longer appear.